src/ol/Graticule.js

```diff
@@ -71,6 +71,9 @@ const INTERVALS = [
  * @property {number} [latLabelPosition=1] Latitude label position in fractions
  * (0..1) of view extent. 0 means at the left of the viewport, 1 means at the
  * right.
+ * @property {function(module:ol/extent~Extent):module:ol/extent~Extent} [getExtent] An optional function
```

Comment:
Better to name the option `extent`, and accept a fixed extent in addition to a function.